src/components/modals/CreateGroupsOrTypesModal.svelte

```diff
@@ -0,0 +1,210 @@
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New Derivation Group Name'}
+                />
+                <select
+                  bind:value={newTypeSourceType}
+                  on:change={() => (newTypeError = null)}
+                  name="newTypeSourceType"
+                  class="st-select"
+                  style="width: 200px"
+                >
+                  {#each $externalSourceTypes as sourceType}
+                    <option value={sourceType.name}>{sourceType.name}</option>
+                  {/each}
+                </select>
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateDerivationGroup}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+            <TabPanel>
+              <p style="padding-top:10px">Provide a name for the new external source type.</p>
+              <p><i>The newly created external source type will be empty, though you can upload sources into it.</i></p>
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New External Source Type Name'}
+                />
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateExternalSourceType}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+            <TabPanel>
+              <p style="padding-top:10px">Provide a name for the new external event type.</p>
+              <p><i>The newly created external event type will be empty, though you can upload events into it.</i></p>
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New External Event Type Name'}
+                />
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateExternalEventType}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+          </Tabs>
+        </div>
+        <div style="flex-direction: column;">
+          <p>
+            <AlertError class="m-2" style="margin: 1.0rem 0; padding-bottom: 0px" error={newTypeError} />
```

Comment:
Is it intended that this error should be presented in both the modal and the upload a source file area?
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/4f26f37d-23a6-4633-a247-bd85365b39cc">

Reply:
I think it should only be shown in the modal itself as the action is occurring from there (i.e., the error doesn't have anything to do directly with the 'Upload a Source File' panel anymore). Updated to remove the errors from that panel!